pkg/server/handler/v1alpha1/workflowrun.go

```diff
@@ -353,7 +353,14 @@ func receiveContainerLogStream(tenant, project, workflow, workflowrun, stage, co
 }
 
 // GetContainerLogStream gets real-time logs of a certain stage.
-func GetContainerLogStream(ctx context.Context, project, workflow, workflowrun, tenant, stage string) error {
+func GetContainerLogStream(ctx context.Context, project, workflow, workflowrun, tenant, stage, qt string) error {
+	// If there is no tenant info in Header, we will get it from query, since front end is hard to
```

Comment:
This is a little tricky, we can use HTTP request and upgrade to websocket. @qiuxiaolian

Reply:
This problem can be solved by setting a proxy. However, the version of the project's create-react-app is too low, and modifying the configuration is cumbersome and requires a relatively large price. Can be resolved after subsequent upgrades